src/content/docs/cheerp/12-reference/10-command-line-options/10-targets.md

```diff
@@ -1,10 +1,25 @@
 ---
-title: WebAssembly output
+title: Targets
+description: WebAssembly, JavaScript, and WASI
 ---
 
-# Generate WebAssembly output
+Cheerp can build applications into the following targets:
 
-Setting the command line flag `-target cheerp-wasm` lets Cheerp generate binary WebAssembly output:
+- [WebAssembly + JavaScript](#webassembly--javascript) (`cheerp-wasm`) (default)
+- [WASI](#webassembly-only-wasi) (`cheerp-wasm-wasi`)
+- [JavaScript](#javascript-only) (`cheerp`)
+
+## WebAssembly + JavaScript
+
+The default target, `-target cheerp-wasm`, produces both a JavaScript file and a WebAssembly file.
+
+The JavaScript file includes all C++ code compiled to JavaScript (code with the `[[cheerp::genericjs]]` attribute), and the code necessary to load the WebAssembly module and run the program.
+
+You can specify a different name and runtime path for the wasm file with the `-cheerp-secondary-output-file=<file>` and `-cheerp-secondary-output-path=<path>` options respectively.
+
+For debugging purposes, human-readable WebAssembly output can be generated using `-cheerp-linear-output=wasm`.
```

Comment:
Did you mean "human-readable asm.js output can be generated using `-cheerp-linear-output=asmjs`" ?

Reply:
This text was there before. I can change this to what you suggest if you prefer. Does `-cheerp-linear-output=wasm` not exist, then?

Reply:
it does exist, but it's the default